.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: rust
 sudo: false
 rust:
+  - stable
```

Comment:
I think it would be nice to explicitly test with 1.15.0 to guarantee compatibility.